css-view-transitions-2/Overview.bs

```diff
@@ -983,6 +1080,56 @@ When capturing the old or new state for an element, perform the following steps
 	Note: This is written in a monkey-patch manner, and will be merged into the algorithm once the L1 spec graduates.
 </div>
 
+## Capturing and applying 'view-transition-group' ## {#vt-group-algorithm}
+
+<div algorithm="additional old apture steps (group)">
+When capturing the old or new state for an element, perform the following steps given a [=captured element=] |capturedElement|, an [=element=] |element| and a {{ViewTransition}} |transition|:
+
+	1. Let |containingGroupName| be the [=used value=] of |element|'s ''view-transition-group''.
+
+	1. If |containingGroupName| is not null, then:
+
+		1. Let |groupContainerElement| be |transition|'s [=ViewTransition/named elements=][containingGroupName].
+
+		1. If |groupContainerElement| is not null, then:
+
+			1. Let |containerElement| be the old element that is being captured for |containingGroupName|.
+
+			1. If |containerElement| is an ancestor of |element|, then:
+
+				1. Set |capturedElement|'s [=captured element/old transform=] to the transform that would map |element|’s [=border box=] from | groupContainerElement|'s [=border box=] to its current visual position.
+
+				1. Set |capturedElement|'s [=captured element/containing group=] to |containingGroupName|.
+</div>
+
+<div algorithm="additional new capture steps (group)">
+When capturing the new state for an element, perform the following steps given a [=captured element=] |capturedElement| and an [=element=] |element|:
+
+	1. Set |capturedElement|'s [=captured element/containing group=] to the [=used value=] of |element|'s 'view-transition-group'.
+</div>
+
+<div algorithm="additional pseudo-element setup steps (group)">
+When setting up the transition pseudo-element for a [=captured element=] |capturedElement|, given a |transitionName| and a {{ViewTransition}} |transition|:
```

Comment:
For this one, could we do 2 sections for each algorithm that needs to be updated?

- https://drafts.csswg.org/css-view-transitions-1/#setup-transition-pseudo-elements-algorithm where we construct the tree once.
- https://drafts.csswg.org/css-view-transitions-1/#style-transition-pseudo-elements-algorithm where we compute the new element's transform each frame.

Reply:
Done